### ISSUE TITLE
Slow clearence level aggregation

### ISSUE THREAD

**Issue by originalsouth** [minvws/nl-kat-coordination#2171]
On a up to date Archlinux (with linux-zen kernel) perform the following:

1. Fresh install (`make reset && make kat`)
1. Add 'Test/0' organization
1. Enable the Wappalizer boefje
1. Add url-ooi pointing to https://mispo.es (this should create a Hostname and HostnameURL ooi)
1. Set the clearance level of the url-ooi to two paws
1. Bug: wait ~10 minutes before Hostname and HostnameURL oois inherit the set clearance level

**Expected behavior**
Wait less than a minute for Hostname and HostnameURL oois to inherit the set clearance level

**Initial analysis**
It seems that during the waiting time is due to Celery is taking a whole CPU after the KAT install (which is possibly related to this [upstream bug](https://github.com/celery/celery/issues/8306))
When during this time ooi clearence level inheritance seems to be suspended.
![image](https://github.com/minvws/nl-kat-coordination/assets/5300799/d3f2129f-3b60-4f5b-8102-8b5c48465a41)
In the logs obtained by `docker compose logs -f octopoes_api_worker rabbitmq` rabbitmq reports messages like:
```
rabbitmq-1             | 2023-12-20 12:54:30.921841+00:00 [warning] <0.1379.0> closing AMQP connection <0.1379.0> (172.28.0.11:46772 -> 172.28.0.6:5672, vhost: 'kat', user: 'eb61296883e76fcdf6af97438ab1263c88fa2f1e2c5ccbcf87'):
rabbitmq-1             | 2023-12-20 12:54:30.921841+00:00 [warning] <0.1379.0> client unexpectedly closed TCP connection
rabbitmq-1             | 2023-12-20 12:56:15.805856+00:00 [error] <0.1224.0> closing AMQP connection <0.1224.0> (172.28.0.12:57620 -> 172.28.0.6:5672):
rabbitmq-1             | 2023-12-20 12:56:15.805856+00:00 [error] <0.1224.0> missed heartbeats from client, timeout: 60s
rabbitmq-1             | 2023-12-20 12:56:15.816879+00:00 [error] <0.1248.0> closing AMQP connection <0.1248.0> (172.28.0.12:57636 -> 172.28.0.6:5672):
rabbitmq-1             | 2023-12-20 12:56:15.816879+00:00 [error] <0.1248.0> missed heartbeats from client, timeout: 60s
rabbitmq-1             | 2023-12-20 12:56:16.095885+00:00 [error] <0.1273.0> closing AMQP connection <0.1273.0> (172.28.0.12:57654 -> 172.28.0.6:5672):
rabbitmq-1             | 2023-12-20 12:56:16.095885+00:00 [error] <0.1273.0> missed heartbeats from client, timeout: 60s
```
This keeps on going until Celery calmes down (which takes to ~10 minutes) after which the inheritance properly commences.
In the logs this is eventually manifested (after many errors) but does not fit this report.
The wait eventually yields the desired behavior:
![image](https://github.com/minvws/nl-kat-coordination/assets/5300799/6f5a1845-b770-4a22-916e-1f36589d1586)

**OpenKAT version**
Tested on 18ee8e616bc0036a296d2b2b7d202d40bc2da075 and older versions going back to October 2023.

**Comment by originalsouth** [comment]
Found the root cause, see https://github.com/celery/billiard/issues/399

**Comment by originalsouth** [comment]
Addressed in #2327.
